error/s2n_errno.h

```diff
@@ -120,6 +119,8 @@ typedef enum {
     S2N_ERR_MISSING_EXTENSION,
     S2N_ERR_UNSUPPORTED_EXTENSION,
     S2N_ERR_DUPLICATE_EXTENSION,
+    S2N_ERR_PQ_DISABLED,
+    S2N_ERR_PQ_CRYPTO,
```

Comment:
I'm not sure either of these should be protocol errors. S2N_ERR_PQ_DISABLED should probably be a usage error, and S2N_ERR_PQ_CRYPTO should be an internal error.